internal/domain/validate/link_test.go

```
@@ -44,11 +44,11 @@ func TestContainsLinkCatches(t *testing.T) {
 		"user @ host . com",
 		"e x a m p l e . c o m",
 		// glyph / invisible (ASCII \u escapes, no raw invisibles in source)
-		"ｅｘａｍｐｌｅ.ｃｏｍ", // full-width example.com
-		"exa​mple.com", // zero-width space
-		"exam‌ple.com", // zero-width non-joiner
+		"ｅｘａｍｐｌｅ.ｃｏｍ",       // full-width example.com
+		"exa​mple.com",      // zero-width space
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode format character U+200B, consider using the '\u200b' escape sequence instead

(staticcheck)